app/templates/createquiz.html

```diff
@@ -0,0 +1,44 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../static/content/createstyle.css">
+{% endblock %}
+{% block content %}
+<div class="create-class-wrapper">
+    <div class="create-container">
+        <form action="" method="post" novalidate class="create-form">
+            {{ form.hidden_tag() }}
+            <h1>Create A New Quiz</h1>
+            {{ form.topic(placeholder="Select Topic", class_="center") }}
+            {{ form.qn(size=32, placeholder="Input Question") }}
+            {{ form.op1(size=32, placeholder="Option 1") }}
+            {{ form.op2(size=32, placeholder="Option 2") }}
+            {{ form.op3(size=32, placeholder="Option 3") }}
+            {{ form.op4(size=32, placeholder="Option 4") }}
+            {{ form.corrOp(placeholder="Correct Option") }}
+            {{ form.img }}
+            <a href="{{ url_for('createquiz') }}" class="route2-btn">Add Another Question</a>
+            <a href="{{ url_for('createquizsuccess') }}" class="route2-btn">Submit Quiz</a>
```

Comment:
No `form.submit`? Are there 2 different submit buttons in the same `CreateQuiz` form?

Reply:
yes, one is to create another question in the quiz, another is to confirm all the previous questions as one quiz